TYPO3.TYPO3CR/Classes/TYPO3/TYPO3CR/Domain/Model/NodeType.php

```diff
@@ -125,6 +125,11 @@ public function __construct($name, array $declaredSuperTypes, array $configurati
         }
 
         $this->localConfiguration = $configuration;
+
+        $this->fullConfiguration = $fullConfiguration;
```

Comment:
seems odd setting it to null when there's a check it could be in if you ask me, but not important ofc.